src/pos.rs

```diff
@@ -137,23 +138,27 @@ impl fmt::Display for Location {
 impl Location {
     pub fn display_with_file(
         &self,
-        file_name_chart: &BTreeMap<u64, String>,
+        file_name_chart: &BTreeMap<u64, FileInfo>,
         line_break: bool,
     ) -> String {
         format!(
             "{}{}In file: {}",
             self,
             if line_break { "\n" } else { " " },
-            file_name_chart
+            /*file_name_chart
```

Comment:
This comment block should probably be deleted.

Reply:
Deleted!